src/components/NewClusterModal.js

```diff
@@ -149,6 +150,66 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     ]))
   }
 
+  updateCluster(isStopRequired = false) {
+    const { currentCluster, onSuccess } = this.props
+    const { googleProject, clusterName } = currentCluster
+
+    if (isStopRequired) {
+      notify('success', 'To be updated, your runtime will now stop, and then start.', { timeout: 3000 })
```

Comment:
TOL: 3 seconds feels a little short for this. What would you think of 5 seconds? Or even not auto-dismissing in this case?